fedn/fedn/common/tracer/mongotracer.py

```diff
@@ -57,11 +57,9 @@ def new_session(self, id=None):
         data = {'session_id': str(id)}
         self.sessions.insert_one(data)
 
-    def new_round(self, id):
+    def new_round(self, round_data):
```

Comment:
can't we call it "create_round" instead, same for session?

Reply:
yes, fixed